### PR TITLE
ksud: remove tailing "/" from `PATH`

### PR DESCRIPTION
**Pull request by skbeh**

### CODE REVIEW

**Comment by tiann** [comment]
tailing with `/` doesn't work?

**Comment by skbeh** [comment]
It works, but it isn't the common practice.

**Comment by tiann** [comment]
I someone edit `BINARY_DIR` someday, error will happen. There isn't any mechanism to ensure this is correct. But if you leave it as it is, it would always work.

File: userspace/ksud/src/module.rs [hunk 1]
```diff
@@ -44,7 +44,11 @@ fn exec_install_script(module_file: &str) -> Result<()> {
         .env("ASH_STANDALONE", "1")
         .env(
             "PATH",
-            format!("{}:{}", env_var("PATH").unwrap(), defs::BINARY_DIR),
+            format!(
+                "{}:{}",
+                env_var("PATH").unwrap(),
+                &defs::BINARY_DIR[..(defs::BINARY_DIR.len() - 1)] // remove tailing "/"
```
Comment (tiann) [inline userspace/ksud/src/module.rs:50]: `trim_end_matches` would be better

**Review by tiann — CHANGES_REQUESTED** [review]

**Review by tiann — APPROVED** [review]

**Pull request merged**